common/laws/00_citizenship.txt

```diff
@@ -263,10 +303,12 @@ law_true_multicultural = {
 		recalculate_pop_ig_support = yes
 	}
 	modifier = {
-		state_radicals_from_sol_change_accepted_culture_mult = 0.5
-		state_loyalists_from_sol_change_accepted_culture_mult = -0.25
-		country_infamy_decay_mult = -0.1
+		state_radicals_from_sol_change_accepted_culture_mult = 0.40
+		state_loyalists_from_sol_change_accepted_culture_mult = -0.20
+		country_infamy_decay_mult = -0.15
```

Comment:
If I made the nationalism infamy increase generation as opposed to decrease decay, would this resolve the discrepancy here?

Reply:
Sort of, it is interesting way to interpret things. It would also fit with foreign laws were only expansionists have a decay modifier, because of the presumed meddling they do with their conquered territories.

Reply:
I'm hoping it will help differentiate what increases generation and what decreases decay, yea. If that's the route for it being more consistent i'll do that.